scipy/optimize/tests/test_zeros.py

```diff
@@ -58,5 +59,17 @@ def test_deriv_zero_warning(self):
         assert_warns(RuntimeWarning, cc.newton, func, 0.0, dfunc)
 
 
+class TestRootResults:
```

Comment:
It's not required (and doesn't work with generator tests anyway, but that's a separate issue) --- nose is required to run the tests.